manifest.webapp

```diff
@@ -64,6 +64,16 @@
       "type": "io.cozy.settings",
       "verbs": ["GET", "PUT"]
     },
+    "profiles": {
+      "description": "Required to manage profiles on Pass extension",
+      "type": "com.bitwarden.profiles",
+      "verbs": ["ALL"]
+    },
+    "folders": {
+      "description": "Required to manage folders on Pass extension",
+      "type": "com.bitwarden.folders",
```

Comment:
I'm wondering if this is still necessary as we don't show bitwarden's folder anymore (current "folders" are just a rewording of "organisations" and "collections")

But maybe we need this to keep compatibility with class bitwarden clients?

Reply:
Even if we don't show the folders, I think the bitwarden clients still make some requests that require this permission. I don't remember exactly, maybe something like the realtime.